app/src/components/exploreFurther/exploreFurther.tsx

```diff
@@ -49,13 +50,20 @@ const ExploreFurther = () => {
             data-testid={`test-id-${index}`}
             mainActionLink={item.url}
             imageProps={{
-              alt: "",
-              aspectRatio: "sixteenByNine",
-              component: undefined,
+              component: (
+                <Image
+                  src={`/${item.image}`}
+                  alt={""}
+                  width={225}
+                  height={126}
```

Comment:
For local images, the next `Image` should be able to determine the width and height automatically, so you shouldn't need these. I'm not sure about "sizes" or "style" though. I'm assuming it's for the aspect ratio?

Reply:
I'm not sure why, but it doesn't recognize these images as public, so there's an error if I remove these props

Reply:
Ah, it's because the image file need to be imported: https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

I think it's worth trying this to make use of the optimization but I think it'll make the loop trickier to work with.

Reply:
Lmk what you think on this approach with the image added to the `exploreFurtherData`